### DISCUSSION TITLE
[RFC] `preserveInvalidOnBlur` for invalidate input keep

### DISCUSSION THREAD

**Discussion by zombieJ** [ant-design/ant-design#46468] [RFCs]
# 📋 RFC

## Summary

For input type component (like DatePicker, InputNumber. etc), provide `preserveInvalidOnBlur` prop to enable not modify value on the input field.

## Basic example

```tsx
<DatePicker preserveInvalidOnBlur />

// Now type `asd` which is not a valid date and blur the input, `asd` will keep in the input
```

## Motivation

For a11y requirement, it should not modify input by default since some user can not notice the value change after blur.

## Detailed

* DatePicker support `preserveInvalidOnBlur`
* InputNumber align the `changeOnBlur={false}` API to `preserveInvalidOnBlur`

## Drawbacks

* InputNumber need update the API

## Adoption strategy

Update in feature branch

**Comment by afc163** [comment]
+1
